examples/directml/llm/llava_model.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2023 the HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
do we need update the license? if the file is copied from hf? could we point the source location?

Reply:
I just looked at the HF transformer code and cannot find the similar file. I suppose this file is copied from somewhere else?

Reply:
It's from here: https://github.com/huggingface/transformers/blob/main/src/transformers/models/llava/modeling_llava.py

It's a stripped-down version which just what's needed to convert the model.